README.md

```diff
@@ -1,8 +1,111 @@
-# apps
-Proof of concept applications that use the Force Move Game framework.
-
-### Setting up development environment and running the app
-* In the top directory, run `yarn install`.
-* In the top directory, run `npx lerna bootstrap`.
-* Run the wallet via `yarn start` in the `wallet` package directory
-* Run the app via `yarn start` in the app package directory.
+<h1 align="center">
+<div><img src="./orange_fireball.svg"> </div>
+Magmo Apps
+</h1>
+Welcome to the Magmo mono-repo, home of several proof-of-concept applications built on our state channel protocols.
+
+## For more information
+On our [website](https://magmo.com) you will find links to our whitepapers and contact information. Whether you simply want to try
+out our apps, or get involved more deeply we would love to hear your thoughts. Deployed versions of our games may be accessed with these links:
+
+* [Rock Paper Scissors](https://demo.magmo.com) (RPS)
+* [Tic Tac Toe](https://ttt.magmo.com) (TTT)
+
+## Setting up development environment and running a game application
+You will need `yarn` installed (see [here](https://yarnpkg.com/lang/en/docs/install/) for instructions). After cloning the code, 
+1. In the top directory, run `yarn install`.
+2. In the top directory, run `npx lerna bootstrap`.
```

Comment:
This can be run in any subdirectory as well.

Reply:
+1

Reply:
👍